o4/o4.py

```diff
@@ -566,15 +609,22 @@ class LogAndAbort(Exception):
                     continue
                 res_str = res.get('depotFile') or res.get('fromFile')
                 if not res_str and res.get('data'):
-                    res_str = head + '/' + res['data']
+                    res_str = client_path_to_depot_path(res['data']) or f'{head}/{res["data"]}'
+
                 if not res_str:
                     errs.append(res)
                     continue
                 res_str = Pyforce.unescape(res_str)
                 for i, f in enumerate(fstats):
-                    if f"{head}/{f[F_PATH]}" in res_str:
-                        repeats[f"{head}/{f[F_PATH]}"].append(res)
-                        not_yet.append(fstats.pop(i))
+                    path = f'{head}/{f[F_PATH]}'
+                    if path in res_str:
```

Comment:
`if res_str.endswith(path)` ?

Reply:
If anything, I'd think it should be ==, since head makes it an absolute depot path